packages/interflex/src/translate.ts

```diff
@@ -0,0 +1,110 @@
+const PLACEHOLDER = /\{(\w+)\}/g;
+const FRAGMENT = /<(\w+)>(.*?)<\/\1>/g;
+const DEFAULT_DICTIONARY = {
+  localizedName: '',
+  currencySymbol: '',
+  thousandSeparator: '',
+  decimalSeparator: '',
+  decimalPlaces: 0,
+  dateFormat: '',
+  strings: {},
+  monthNames: [],
+  monthNamesShort: [],
+  daysOfWeek: [],
+  daysOfWeekShort: [],
+  startOfWeek: 0,
+};
+let dictionary: LocaleDictionary = {};
+let tLocale: Locale = 'en';
```

Comment:
I really hesitate to use a language without a region because there are so many differences between `en_US` and `en_AU` such as "colour" and `licence` and I'm sure the same is true between Canadian French and France, etc.